File: showly-react-native/src/core/utils/cleanHtml.ts
export function cleanHtml(value?: string | null): string {
  if (!value) return "No summary available.";

  return value
    .replace(/<[^>]*>/g, "")
    .replace(/&nbsp;/g, " ")
    .replace(/&amp;/g, "&")
    .replace(/&quot;/g, '"')
    .trim();
}

File: showly_flutter/lib/core/utils/clean_html.dart
String cleanHtml(String? value) {
  if (value == null || value.trim().isEmpty) {
    return 'No summary available.';
  }

  return value
      .replaceAll(RegExp(r'<[^>]*>'), '')
      .replaceAll('&nbsp;', ' ')
      .replaceAll('&amp;', '&')
      .replaceAll('&quot;', '"')
      .trim();
}
